### PR TITLE
feat/readme-config

### PR DESCRIPTION
**Pull request by khenriksson**
Made the appropriate changes from the #62, mentioning the change of `clockify config init` to `clockify-cli config --init`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,8 +59,59 @@ go get -u github.com/lucassabreu/clockify-cli
...
+```
+
+After finishing your work you can stop the entry using `clockify-cli out`
+
+```console
+foo@bar:~$ clockify-cli out
++------------------+----------+----------+---------+---------+-------------------+------+
+|        ID        |  START   |   END    |   DUR   | PROJECT |    DESCRIPTION    | TAGS |
++------------------+----------+----------+---------+---------+-------------------+------+
+|    time-entry-id    | 16:02:45 | 16:08:06 | 0:05:21 |         | Clockify CLI Test |      |
```
Comment (lucassabreu) [inline README.md:98]:
```suggestion
+------------------+----------+----------+---------+---------+-------------------+------+
|   time-entry-id  | 16:02:45 | 16:08:06 | 0:05:21 |         | Clockify CLI Test |      |
```

File: README.md [hunk 2]
```diff
@@ -62,8 +59,59 @@ go get -u github.com/lucassabreu/clockify-cli
...
+```console
+foo@bar:~$ clockify-cli in -i
+? Choose your project: project-id - Example Project
+? Description: Clockify CLI Test
+? Start: (2020-09-04 16:02:45)
+? End (leave it blank for empty):
++------------------+----------+----------+---------+---------+-------------------+------+
+|        ID        |  START   |   END    |   DUR   | PROJECT |    DESCRIPTION    | TAGS |
++------------------+----------+----------+---------+---------+-------------------+------+
+|    time-entry-id    | 16:02:45 | 16:03:47 | 0:01:02 |         | Clockify CLI Test |      |
```
Comment (lucassabreu) [inline README.md:87]:
```suggestion
|   time-entry-id  | 16:02:45 | 16:03:47 | 0:01:02 |         | Clockify CLI Test |      |
```

File: README.md [hunk 3]
```diff
@@ -62,8 +59,59 @@ go get -u github.com/lucassabreu/clockify-cli
...
+
+```console
+foo@bar:~$ clockify config init
+? User Generated Token: <your-api-token>
+? Choose default Workspace: workspace-id - John Doe's workspace
+? Choose your user: user-id - John Doe
+? Should try to find project by its name? (y/N) Yes
+```
+
+The CLI saves your workspace info and an API token to `~/.clockify-cli.yaml` for future use.
```
Comment (lucassabreu) [inline README.md:74]:
```suggestion
The CLI saves your workspace info and an API token to `~/.clockify-cli.yaml` for future use.

> :exclamation: If you have installed the client using `snap` this file will not be accessible to you, but the configs will still be persisted.
```

**Review by lucassabreu — COMMENTED** [review]
same suggestions a left for @stg101, if you could revert the title and list formatting would also be great.

once done i will approve and merge it.

**Comment by khenriksson** [comment]
Not sure about the list formatting and titles, but I did what I suspected it was.

**Review by lucassabreu — APPROVED** [review]

**Pull request merged**